pkg/executioncluster/impl/random_cluster_selector_test.go

```diff
@@ -32,13 +30,13 @@ const testDomain = "domain"
 const testWorkflow = "name"
 
 func initTestConfig(fileName string) error {
-	var searchPaths []string
-	for _, goPath := range strings.Split(build.Default.GOPATH, string(os.PathListSeparator)) {
-		searchPaths = append(searchPaths, filepath.Join(goPath, "src/github.com/lyft/flyteadmin/pkg/executioncluster/testdata/", fileName))
+	pwd, err := os.Getwd()
+	if err != nil {
+		return err
 	}
 
 	configAccessor := viper.NewAccessor(config.Options{
-		SearchPaths: searchPaths,
+		SearchPaths: []string{filepath.Join(pwd, "../testdata", fileName)},
```

Comment:
Thank you so much, but what do you think about moving the test yaml to this folder?

Reply:
does not need to part of this change, we can file a bug and handle it as a follow up.

Reply:
Yeah I prefer that in a separated PR because it is not really part of this fix.